pkg/sync/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/testground/testground/pkg/sync
+
+go 1.15
+
+require (
+	github.com/go-redis/redis/v7 v7.4.0
+	github.com/google/uuid v1.1.1
+	github.com/testground/testground v0.5.3
```

Comment:
Why is the `sync` service requiring Testground? We should create a separate repo for the sync service, so that we don't have nested go mods - versioning would be a mess otherwise I think.

Reply:
Hmm... it's just the logging. But it needs to be a different package anyways, because the sdk-go needs to import some types... and testground imports sdk-go.

I'll work on moving it to a different repository then.